lib/dialects/mysql/query-generator.js

```diff
@@ -495,15 +523,17 @@ const QueryGenerator = {
    * @private
    */
   getForeignKeyQuery(table, columnName) {
-    let tableName = table.tableName || table;
-    if (table.schema) {
-      tableName = table.schema + '.' + tableName;
-    }
-    return 'SELECT CONSTRAINT_NAME as constraint_name'
+    const tableName = table.tableName || table;
+
```

Comment:
Remove this line

Reply:
I think it should keep the same as https://github.com/sequelize/sequelize/blob/master/lib/dialects/mssql/query-generator.js#L649

Reply:
I meant the empty line between two const definition

Reply:
Sorry for misunderstanding